go/ephemeral/user_ek_test.go

```diff
@@ -111,11 +111,6 @@ func TestDeviceRevokeNewUserEK(t *testing.T) {
 	userEK, err := tc.G.GetUserEKBoxStorage().Get(context.Background(), 2)
 	require.NoError(t, err)
 	require.Equal(t, secondStatement.CurrentUserEkMetadata, userEK.Metadata)
-	// Repeat those checks after clearing the cache, to test unboxing.
```

Comment:
I realized that clearing the cache isn't necessary (since we don't also purge the copy on disk like other tests), sorry about the back and forth about this before!